### PR TITLE
Add Goofy Switch to Carving Diagram

### PR DESCRIPTION
**Pull request by habdelra**
Adds a goofy switch to the carving diagram. cheers @tintinthong !

image

### CODE REVIEW

File: packages/drafts-realm/carving-turn-diagram.gts [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+ *   the auto-saving during bad syntax), it was annoying that re-indexing wasn't
+ *   triggered after the card definition file change. I suspect that if I was
+ *   viewing the card deifnition in monaco it live updates (i didn't happen to have
+ *   card definition open in monaco at the time). but instances that consume the card
+ *   definition are not live updated when the gts file changes on disk. I think this
+ *   is either an oversigh or perhaps a bug. Not sure that it is that important, though,
+ *   as generally people will not be using VS code to directly edit cards in a local
+ *   boxel instance (I assume).
+ */
+
```
Comment (habdelra) [inline packages/drafts-realm/carving-turn-diagram.gts:37]: FYI: here's my champange feedback

**Comment by habdelra** [comment]
Todo:
- [ ] Position 10 label on the toe turn intersects with board annotation 
- [x] use  the new field descriptions feature
- [x] Add catalog entry

**Review by tintinthong — APPROVED** [review]
🤘🤘 Awesome

**Review by jurgenwerk — APPROVED** [review]

**Comment by IanCal** [comment]
*very* cool viz

**Pull request merged**